golangci.yaml

```diff
@@ -1,12 +1,35 @@
-linters:
-  disable:
-    - megacheck
+liners:
+  disable-all: true
   enable:
+    - bodyclose
+    - deadcode
+    - depguard
+    - dogsled
+    - errcheck
+    - gocyclo
+    - gofmt
+    - goimports
```

Comment:
`goimports` assumes that `make fmt` is using `goimports`.

Reply:
I'm having trouble installing goimports with bingo. I've put it on the commented wish list for now. I'd like to come back to it but I want to get this merge in ASAP as it touches a lot of files and will become a merge-out nightmare quickly.